js/ElementRendererProvider.js

```diff
@@ -490,6 +490,7 @@ export default class ElementRendererProvider {
 
           divCarouselWrapper.appendChild(carousel);
           carousel.className = 'lp-json-pollock-layout-carousel';
+          carousel.setAttribute('aria-label', config.tooltip);
```

Comment:
if no tooltip is provided, it is better not add an empty attribute
```suggestion
          if (config.tooltip) {
            carousel.setAttribute('aria-label', config.tooltip);
          }
```